apps/process_tpstream.cxx

```diff
@@ -139,17 +154,17 @@ int main(int argc, char const *argv[])
   std::string input_file_path;
   app.add_option("-i", input_file_path, "Input TPStream file path")->required();
   std::string output_file_path;
-  app.add_option("-o", output_file_path, "Output TPStream file path")->required();
+  app.add_option("-o", output_file_path, "Output TPStream file path");
   std::string channel_map_name = "VDColdboxChannelMap";
   app.add_option("-m", channel_map_name, "Detector Channel Map");
-  std::string plugin_name = "TriggerActivityMakerHorizontalMuonPlugin";
-  app.add_option("-p", plugin_name, "Trigger Activity plugin name.");
+  // std::string plugin_name = "TriggerActivityMakerHorizontalMuonPlugin";
+  // app.add_option("-p", plugin_name, "Trigger Activity plugin name.");
   std::string config_name;
-  app.add_option("-j", config_name, "Trigger Activity config JSON to use.");
+  app.add_option("-j", config_name, "Trigger Activity config JSON to use.")->required();
```

Comment:
Maybe would be worth to include a default json config, for both TA and TC making?